plugins/modules/zos_operator_action_query.py

```diff
@@ -72,11 +84,16 @@
   zos_operator_action_query:
       message_id: dsi*
 
-- name: Display all outstanding messages given job_name, message_id, system
+- name: Display all outstanding messages that have the text IMS READY in them
+  zos_operator_action_query:
+      msg_filter: ^.*IMS READY.*$
+
+- name: Display all outstanding messages given job_name, message_id, system, msg_filter
```

Comment:
This is also minor:

"Display all outstanding messages given job_name, message_id, system, msg_filter" probably should be tweaked to "Display all outstanding IMS messages given a job_name, message_id and system using msg_filter."